holopy/scattering/calculations.py

```diff
@@ -39,24 +38,27 @@
     from holopy.scattering.theory.dda import DDA
 except:
     pass
+    # FIXME why is this a try-pass?
```

Comment:
It doesn't need to be anymore. It should now be safe to do these imports even if the relevant scattering theories aren't compiled/installed since the error is now raised further downstream.

Reply:
Fixed with commit aa1066d